docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -765,28 +1191,39 @@ module.exports = {
       // [...]
       env: {
         // --- Remember to set the properties below
-        SERVICE_BROKER: 'redis://localhost:6379/0',
-        SERVICE_MARKET_REDIS: 'redis://localhost:6379/2',
-        SERVICE_MARKET_FIAT_CURRENCIES: 'EUR,USD,CHF,GBP,RUB',
-        SERVICE_MARKET_TARGET_PAIRS: 'LSK_BTC,LSK_EUR,LSK_USD,LSK_CHF,BTC_EUR,BTC_USD,BTC_CHF',
+        SERVICE_BROKER: 'redis://127.0.0.1:6379/0',
+        SERVICE_MARKET_REDIS: 'redis://127.0.0.1:6379/2',
+        SERVICE_MARKET_FIAT_CURRENCIES: 'EUR,USD,CHF,GBP,RUB,PLN,JPY,AUD,GBP,INR',
+        SERVICE_MARKET_TARGET_PAIRS: 'LSK_BTC,LSK_EUR,LSK_USD,LSK_CHF,LSK_PLN,LSK_JPY,LSK_AUD,LSK_GBP,LSK_INR,BTC_EUR,BTC_USD,BTC_CHF',
         // EXCHANGERATESAPI_IO_API_KEY: ''
+        // SERVICE_BROKER_TIMEOUT: 10,
+        // SERVICE_LOG_LEVEL: 'info',
+        // SERVICE_LOG_CONSOLE: false,
+        // SERVICE_LOG_STDOUT: true,
+        // SERVICE_LOG_GELF: false,
+        // SERVICE_LOG_FILE: false,
+        // DOCKER_HOST: 'local',
+        // JOB_INTERVAL_REFRESH_PRICES_BINANCE: 0,
+        // JOB_SCHEDULE_REFRESH_PRICES_BINANCE: '* * * * *',
+        // JOB_INTERVAL_REFRESH_PRICES_BITTREX: 0,
+        // JOB_SCHEDULE_REFRESH_PRICES_BITTREX: '* * * * *',
+        // JOB_INTERVAL_REFRESH_PRICES_EXCHANGERATESAPI: 0,
+        // JOB_SCHEDULE_REFRESH_PRICES_EXCHANGERATESAPI: '* * * * *',
+        // JOB_INTERVAL_REFRESH_PRICES_KRAKEN: 0,
+        // JOB_SCHEDULE_REFRESH_PRICES_KRAKEN: '* * * * *',
+        // JOB_INTERVAL_UPDATE_PRICES: 5,
+        // JOB_SCHEDULE_UPDATE_PRICES: '',
       },
     },
     // Configuration for other microservices.
   ],
 };
 ----
---
-=====
+====
 
 === Export
```

Comment:
Can we please re-order the entries to match the following?

- SERVICE_BROKER
- SERVICE_EXPORT_REDIS
- SERVICE_EXPORT_REDIS_VOLATILE
- SERVICE_EXPORT_PARTIALS
- SERVICE_EXPORT_STATIC
- EXPORT_S3_ENDPOINT
- EXPORT_S3_ACCESS_KEY
- EXPORT_S3_SECRET_KEY
- EXPORT_S3_SESSION_TOKEN
- EXPORT_S3_REGION
- EXPORT_S3_BUCKET_NAME
- EXPORT_S3_BUCKET_NAME_PARTIALS
- EXPORT_S3_BUCKET_NAME_STATIC
- JOB_INTERVAL_CACHE_PURGE
- JOB_SCHEDULE_CACHE_PURGE

It'd also be nice to state that user to configure the `EXPORT_S3_*` variables only when they intend to use AWS S3 instead of the local filesystem, which are determined by `SERVICE_EXPORT_PARTIALS` and `SERVICE_EXPORT_STATIC`.